packages/sitecore-jss-react/src/components/File.test.tsx

```diff
@@ -50,4 +50,31 @@ describe('<File />', () => {
     expect(rendered.html()).to.contain('id="my-file"');
     expect(rendered.html()).to.contain('class="my-css"');
   });
+
+  it('should render field metadata when metadata property is present', () => {
+    const testMetadata = {
+      contextItem: {
+        id: '{09A07660-6834-476C-B93B-584248D3003B}',
+        language: 'en',
+        revision: 'a0b36ce0a7db49418edf90eb9621e145',
+        version: 1,
+      },
+      fieldId: '{414061F4-FBB1-4591-BC37-BFFA67F745EB}',
+      fieldType: 'file',
+      rawValue: 'Test1',
+    };
+
+    const field = {
+      src: '/lorem',
+      title: 'ipsum',
+      metadata: testMetadata,
+    };
+
+    const rendered = mount(<File field={field} editable={true} />);
+    console.log(rendered.html());
+    // expect(rendered.find('code')).to.have.length(2);
+    expect(rendered.html()).to.contain('kind="open"');
```

Comment:
@yavorsk Another node. We need to verify that the exact structure that we agreed between us and Pages is rendered.
It's not enough to check that some blocks are rendered and present. The structure should be verified also